model/src/w3iogrmd.F90

```diff
@@ -899,7 +899,7 @@ SUBROUTINE W3IOGR ( INXOUT, NDSM, IMOD, FEXT &
            FLCK, FLSOU, FLBPI, FLBPO, CLATS, CLATIS, CTHG0S,      &
            STEXU, STEYU, STEDU, IICEHMIN, IICEHINIT, IICEDISP,    &
            ICESCALES(1:4), CALTYPE, CMPRTRCK, IICEHFAC, IICEHDISP,&
-           IICEDDISP, IICEFDISP, BTBETA,                          &
+           IICEDDISP, IICEFDISP, BTBETA,IC_NUMERICS,              &
```

Comment:
@sbanihash  can you add this new variable to: https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_grid.inp#L456

Reply:
Thanks @JessicaMeixner-NOAA for your review. New variable is added.